database/datagenerator.js

```diff
@@ -151,6 +151,7 @@ const reviews = function generateReviews(reviewers, restaurants, paragraphs) {
     })
```

Comment:
Not 100% sure what the closing parens and curly bracers on lines 151-153 are about, but perhaps I can't see their starting points.

Reply:
Ha, ha! I see it now! Github had minimized the code so I couldn't see. My bad!